### PR TITLE
Feature/new meme page

### PR DESCRIPTION
**Pull request by TheKingWizard**
I made a cloudinary account to temporarily work with and the images are currently being uploaded there. An upload preset needs to be made to allow unsigned uploads to the actual cloud. I know this isn't good security-wise, but it works for now and we can change it if we want to.

### CODE REVIEW

**Comment by TheKingWizard** [comment]
[Create Upload page](https://trello.com/c/DnE8zlid/16-create-upload-page)

**Review by fnoori — APPROVED** [review]
One little feedback, when the image uploads, it should clear the page or kick the user to a different page.
As for the cloudinary account, we can keep it as you have it, it honestly doesn't really matter where it gets uploaded, as long as it is viewable/retrievable.

File: src/css/app.styl [hunk 1]
```diff
@@ -15,3 +15,32 @@
...
+    position: static;
+    height: inherit;
+    width: 200%;
+    opacity: 0;
+    z-index: 2;
+}
+
+#new-meme-facade {
+    position: relative;
+}
```
Comment (nxtman123) [inline src/css/app.styl]: You can move these into the `<style>` section of `New.vue`

File: src/pages/New.vue [hunk 2]
```diff
@@ -1,12 +1,131 @@
...
+        class="full-width"
+      />
+    </form>
   </q-page>
 </template>
 
 <script>
+// eslint-disable-next-line
+const CLOUD_NAME = 'kingwizard'; // TODO: Channge to correct cloud
+const CLOUD_PRESET = 'h4c8kn5g'; // TODO: Add upload preset to correct cloud
```
Comment (nxtman123) [inline src/pages/New.vue]: Add `CLOUDINARY_NAME` and `CLOUDINARY_PRESET` to `.example.env` and talk about it in the readme.  Keep secrets hidden and out of git, even though I think we're sending it to the client :P

```
const CLOUD_NAME = process.env.CLOUDINARY_NAME
const CLOUD_PRESET = process.env.CLOUDINARY_PRESET
```

**Review by nxtman123 — COMMENTED** [review]
Looks good, just a couple things

**Pull request merged**